jest.config.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
   reporters: ['<rootDir>/tests/reporter.js'],
+  testTimeout: 20000,
```

Comment:
why do we need this?

Reply:
The full-page screenshot takes a bit more time than the panel screenshot so tests are failing without it. The default is 5s, this is setting it to 20s. I think this is reasonable. It could even be more, knowing that the image renderer config timeout is 30s